pepper-apis/config/application.conf.ctmpl

```diff
@@ -98,7 +99,11 @@
         # Batch size to use for checking status and scheduling
         "batchSize": {{$conf.Data.kits.batchSize}}
     },
+    {{if eq $app "housekeeping"}}
+    "jcacheConfigurationFile": "redisson-jcache-housekeeping.yaml",
+    {{else}}
     "jcacheConfigurationFile": "redisson-jcache.yaml",
+    {{end}}
     "redisServerAddress": "redis://{{$conf.Data.redis.host}}:{{$conf.Data.redis.port}}",
```

Comment:
I think this is first time we have a condition in config for housekeeping vs pepper-backend. Thought of generating a separe housekeeping config file but there would be lot of duplication.